flow-typed/ethereumjs-testrpc.js

```diff
@@ -0,0 +1,7 @@
+// @flow
+/* eslint-disable flowtype/no-weak-types */
+
+declare module 'ethereumjs-testrpc' {
+  declare module.exports: any
+  declare type TestRPCParams = any
```

Comment:
Is this really needed?

Reply:
Yes, it's used in TestEthereumClient. Docs of flow say that it's a good practice to define types for untyped external libs